utilities/debug.cc

```diff
@@ -8,9 +8,49 @@
 #include "rocksdb/utilities/debug.h"
 
 #include "db/db_impl/db_impl.h"
+#include "rocksdb/utilities/options_type.h"
 
 namespace ROCKSDB_NAMESPACE {
 
+static std::unordered_map<std::string, ValueType> value_type_string_map = {
```

Comment:
I wonder if we can put assert/static_assert to make sure this map covers all known types.

Reply:
I added a simple unittest to make sure the list is up to date.